renderdoc/os/posix/posix_libentry.cpp

```diff
@@ -42,9 +42,14 @@ void library_loaded()
   string curfile;
   FileIO::GetExecutableFilename(curfile);
 
+#ifdef RENDERDOC_PLATFORM_ANDROID
```

Comment:
The idea of splitting OS files by platform is to avoid profusion of #ifdefs, so if there truly needed to be two implementations I'd prefer this to be split out into different libentry files (with minimal duplication - e.g. expose a function `IsReplayApplication()`). However as I mentioned in the comment above, it seems like the package name could just be returned for GetExecutableFilename, and then we can just add `|| curfile.find(".renderdoccmd")` to the if.